phpstan-baseline.neon

```diff
@@ -5240,6 +5240,11 @@ parameters:
 			count: 1
 			path: src/Sylius/Component/Core/Promotion/Checker/Rule/TotalOfItemsFromTaxonRuleChecker.php
 
+		-
+			message: "#^Method Sylius\\\\Component\\\\Core\\\\Promotion\\\\Checker\\\\TaxonInPromotionRuleChecker\\:\\:__construct\\(\\) has parameter \\$promotionRuleRepository with generic interface Sylius\\\\Component\\\\Resource\\\\Repository\\\\RepositoryInterface but does not specify its types\\: T$#"
```

Comment:
It could be also resolved by a proper docblock in the checker

Reply:
*It **must** be 😂